lms/djangoapps/support/static/support/jsx/file_upload.jsx

```diff
@@ -0,0 +1,170 @@
+/* global gettext */
+/* eslint one-var: ["error", "always"] */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ShowProgress from './upload_progress';
+
+
+class FileUpload extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.uploadFile = this.uploadFile.bind(this);
+    this.removeFile = this.removeFile.bind(this);
+    this.state = {
+      fileList: [],
+      fileInProgress: null,
+    };
+  }
+
+  removeFile(e) {
+    e.preventDefault();
+    const fileToken = e.target.id,
+      $this = this,
+      url = `https://arbisoft.zendesk.com/api/v2/uploads/${fileToken}.json`,
+      accessToken = 'd6ed06821334b6584dd9607d04007c281007324ed07e087879c9c44835c684da',
+      request = new XMLHttpRequest();
+
+    request.open('DELETE', url, true);
+    request.setRequestHeader('Authorization', `Bearer ${accessToken}`);
+    request.setRequestHeader('Content-Type', 'application/json;charset=UTF-8');
+
+    request.send();
+
+    request.onreadystatechange = function removeFile() {
+      if (request.readyState === 4 && request.status === 204) {
+        $this.setState({
+          fileList: $this.state.fileList.filter(file => file.fileToken !== fileToken),
+        });
+      }
+    };
+  }
+
+  uploadFile(e) {
+    const url = 'https://arbisoft.zendesk.com/api/v2/uploads.json?filename=',
+      fileReader = new FileReader(),
+      request = new XMLHttpRequest(),
+      errorList = [],
+      $this = this,
+      file = e.target.files[0],
+      accessToken = 'd6ed06821334b6584dd9607d04007c281007324ed07e087879c9c44835c684da',
+      maxFileSize = 5000000, // 5mb is max limit
+      allowedFileTypes = ['gif', 'png', 'jpg', 'jpeg', 'pdf'];
+
+    // remove file from input and upload it to zendesk after validation
+    $(e.target).val('');
+
+    if (file.size > maxFileSize) {
+      errorList.push(gettext('Please select a file with a size less than 5 MB.'));
```

Comment:
@tasawernawaz  please change this message text to: "Files that you upload must be smaller than 5MB in size."